vector/src/main/java/im/vector/RegistrationManager.java

```diff
@@ -890,15 +890,15 @@ public void onNetworkError(final Exception e) {
 
                         @Override
                         public void onUnexpectedError(Exception e) {
-                            listener.onThreePidRequested(pid);
+                            listener.onThreePidRequestFailed(R.string.account_email_error, e.toString());
```

Comment:
Correct code should be:
`listener.onThreePidRequestFailed(R.string.account_phone_number_error, e.getLocalizedMessage());`